test/OpenTelemetry.Extensions.Tests/ActivityEventAttachingLogProcessorTests.cs

```diff
@@ -180,4 +191,61 @@ public void Dispose()
             Assert.Empty(activity.Events);
         }
     }
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    [InlineData(true, 18, true, true, true)]
+    [InlineData(true, 0, false, false, true, true)]
+    public void AttachLogsToActivityEventTest_Filter(
+        bool sampled,
+        int eventId = 0,
+        bool includeFormattedMessage = false,
+        bool parseStateValues = false,
+        bool includeScopes = false,
+        bool recordException = false)
+    {
+        this.sampled = sampled;
+
+        using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.IncludeScopes = includeScopes;
+                    options.IncludeFormattedMessage = includeFormattedMessage;
+                    options.ParseStateValues = parseStateValues;
+                    options.AttachLogsToActivityEvent(x => x.Filter = _ => false);
```

Comment:
what does this test case intend to test?

Reply:
The idea was to test that whatever configuration is set, all get filter-out

Reply:
So it has the same structure as the `AttachLogsToActivityEventTest`, but it checks there is no logging with `filter=>false`

Reply:
I suggest to add dedicated test to cover the new functionality.
1. Test that filter -> true does attach log as activity event
2. Test that filter -> false does not attach log as activity event.
3. Test that filter -> exception does not attach log as activity event, and does not affect the rest of logging/tracing pipeline.

Reply:
Test covers throwing filter